src/scanner/sky.go

```diff
@@ -0,0 +1,215 @@
+// Package scanner scans bitcoin blockchain and check all transactions
+// to see if there are addresses in vout that can match our deposit addresses.
+// If found, then generate an event and push to deposit event channel
+//
+// current scanner doesn't support reconnect after btcd shutdown, if
+// any error occur when call btcd apis, the scan service will be closed.
+package scanner
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/skycoin/skycoin/src/api/webrpc"
+	"github.com/skycoin/skycoin/src/util/droplet"
+	"github.com/skycoin/skycoin/src/visor"
+)
+
+// SKYScanner blockchain scanner to check if there're deposit coins
+type SKYScanner struct {
+	log       logrus.FieldLogger
+	skyClient SkyRPCClient
+	// Deposit value channel, exposed by public API, intended for public consumption
+	Base CommonScanner
+}
+
+// SkyClient implements the SKYRpcClient interface
+type SkyClient struct {
+	c *webrpc.Client
+}
+
+// NewSkyClient create a new skyclient instance
+func NewSkyClient(addr string) *SkyClient {
+	skyClient := SkyClient{
+		c: &webrpc.Client{Addr: addr},
```

Comment:
Use the gui.Client instead, you'll need to update Gopkg.toml to use the v0.22.0 tag in the dependencies